Oracle.md

```diff
@@ -113,12 +113,30 @@ If the rate is greater than $999999, the oracle signs the digits `9`, `9`, `9`,
 
 ### Serialization and signing of outcome values
 
-Every outcome value should be encoded as [a UTF-8 string with NFC normalization](./Messaging.md#Fundamental-types), before being passed to the signing algorithm.
+Every outcome value (`outcome_value`) should be encoded as [a UTF-8 string with NFC normalization](./Messaging.md#Fundamental-types), before being passed to the signing algorithm.
```

Comment:
nit: I'm not sure why you add `outcome_value` here since you define it below

Reply:
Figured it is more clear